<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_pesquisarproduto</class>
 <widget class="QDialog" name="fm_pesquisarproduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pesquisar Produto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Arquivo_de_recursos.qrc">
    <normaloff>:/imagens/img/procurar.png</normaloff>:/imagens/img/procurar.png</iconset>
  </property>
  <widget class="QTableWidget" name="tw_listaProdutos">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>130</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>129</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Pesquisar por:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rb_idProduto">
        <property name="text">
         <string>ID Produto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_nomeProduto">
        <property name="text">
         <string>Nome Produto</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_confirmar">
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>44</y>
     <width>152</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="le_pesquisar"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pesquisar">
      <property name="text">
       <string>Pesquisar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Arquivo_de_recursos.qrc"/>
 </resources>
 <connections/>
</ui>
